libc/config/linux/x86_64/config.json

```diff
@@ -0,0 +1,8 @@
+{
+    "setjmp": {
+        "LIBC_CONF_SETJMP_FORTIFICATION": {
+            "value": true,
+            "doc": "Protect jmp_buf by masking its contents and storing a simple checksum, to make it harder for an attacker to read meaningful information from a jmp_buf or to modify it. This is only supported on x86-64 Linux."
```

Comment:
Perhaps this should be one config. Otherwise, if the user flips one but not the other, then setjmp/longjmp will fail at runtime, right?